### DISCUSSION TITLE
User Experience Options

### DISCUSSION THREAD

**Discussion by arrowgent** [SebLague/Geographical-Adventures#16] [Ideas]
with watching your dev videos and seeing what your ideas were and how youre building
- 
as a "game" i think it would be great for a sandbox of options for the player to have (see Universe Sandbox 2)
as i played i was comparing this to GeoGuesser or surfing around on google earth map

* toggle day night, slider for length of cycle
* enable country names as you over on them
* sliders for various effects on the globe: water on/off, colors, night light colors, etc.
* speed options like 0-10 where 1 or 5 is default speed

so many options could be allowed for a sandbox gameplay style where the player chooses how to experience the gameplay
however "default" out of the box could be very bare minimum to stylize the gameplay, make it challenging and engaging.

--

further:  scorekeeping.
for each country (not all 7400 cities) keep a highscore record for nearest drop in that country
score could be based on time and distance.  since _perfect_ scores everytime would basically be futile for having scores

--

as for SFX and MUSIC.  i think a standard CCSA quiet soft background track would be acceptable, but i dont see why someone cant play whatever music they want over it.  yet i still get why people see the need to request music.
SFX:  not much needed other than a dropping sfx and a DING you win or clapping sfx.  something really basic that is fun but not obnoxious
any music, sfx, or model / graphics considerations must be weighed heavily with licensing and "copyrights" but there are valid options out there

--

i know majority of your efforts are on the technical data points and programming
but also some choices in character model for the airplane.  if someone wants to contribute some designs
cant wait to see your cloud effects, adding the ships back in, and other features
i was already amazed you even bothered to make the moon precession on its axis, as many people would A) never notice or B) dont know enough astronomy to understand these subtle changes in the "bell rocking" of the moon's orbit

**Comment by arrowgent** [comment]
oh, i remembered one idea:

when you drop the package, if over a distance lets say "256km" then show a line from the drop point to the city, something noticable like a red, blue, or green line ending with an arrow
this can help direct, orientation, and teaching where places are located

if under 256km, the line wouldnt be as useful since it should be somewhere visible on the screen already
"256km" is an estimate and isnt important, you could draw a line if the package was 50km away, depending on the size of the line from the drop point.

**Comment by SebLague** [comment]
Hey, thanks for sharing your thoughts, I appreciate it!

**Comment by dis-da-mor** [comment]
Those are great ideas, I also have a few suggestions:

-borders are a little hard to see, especially at night, so maybe have them glow?
-have a mini map on screen so that you can tell where you are on the globe roughly
-markers that can point to the country where there's a package to pick up / drop off, this could maybe be set on the map via a double click
